server/utils/applicationInsightsUrlBuilder.ts

```diff
@@ -11,4 +11,11 @@ const buildUrl = (query: string, timespan = 'PT30M') => {
   )}/timespan/${timespan}`
 }
 
-export default buildUrl
+export const buildUrlNoTimespan = (query: string) => {
+  const resource = encodeURIComponent(
+    `/subscriptions/${config.applicationInsights.subscriptId}/resourceGroups/${config.applicationInsights.resourceGroup}/providers/Microsoft.Insights/components/${config.applicationInsights.component}`,
+  )
+  return `${
+    config.applicationInsights.url
+  }/resourceId/${resource}/source/LogsBlade.AnalyticsShareLinkToQuery/query/${encodeURIComponent(query)}`
+}
```

Comment:
Adding an alternative App Insights URL builder that doesn't include a timespan (e.g. last 24 hours) - it's not needed because we have the exact start/end dates of the migration and it just confused users when they couldn't see anything